test/demo/utils/globalSetup.ts

```diff
@@ -5,14 +5,29 @@ import * as path from 'path';
 import * as mkdirp from 'mkdirp';
 import fetch from 'node-fetch';
 import * as puppeteer from 'puppeteer';
+import * as jsAdapter from 'hadouken-js-adapter';
 
 declare const global: NodeJS.Global&{__BROWSER_GLOBAL__: puppeteer.Browser};
 
 const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup');
 
 module.exports = async () => {
-    // Openfin should be running at this point
-    const browserWSEndpoint = (await (await fetch('http://localhost:9222/json/version')).json()).webSocketDebuggerUrl;
+    // Get the devtools port of the currently running openfin instance
```

Comment:
I figured I would pull these improvements in from Notifications, since it seems to improve the connection errors at least a bit.